dsalgo/graphs.py

```diff
@@ -0,0 +1,66 @@
+# Python3 Program to print BFS traversal 
+# from a given source vertex. BFS(int s) 
+# traverses vertices reachable from s. 
```

Comment:
Maybe we don't need these comments above import lines. What do you think @codesankalp